src/bundle/DependencyInjection/Compiler/DriverPass.php

```diff
@@ -42,24 +42,21 @@ public function process(ContainerBuilder $container)
         $container->setAlias(ContentTagInterface::class, 'fos_http_cache.http.symfony_response_tagger');
     }
 
-    public static function getTaggedService(ContainerBuilder $container, $tag)
+    public static function getTaggedService(ContainerBuilder $container, $tag): string|null
```

Comment:
```suggestion
    public static function getTaggedService(ContainerBuilder $container, $tag): ?string
```

Reply:
Is this function being public intentional? It seems to be part of this CompilerPass internals tbh.